src/worldInteractions.ts

```diff
@@ -318,7 +320,7 @@ class WorldInteraction {
         }
       }
       // todo placing with offhand
-      if (cursorBlock && !activate && !stop) {
+      if (!inAdventure && cursorBlock && !activate && !stop) {
```

Comment:
@Mqxx oh i missed this part. This is not correct since it also disables block interactions (like crafting table, buttons!), you are free to help implement this fix in github.com/zardoy/mineflayer-mouse